sensorconnect_plugin/downloadPortModeData_test.go

```
@@ -37,14 +38,23 @@
-				Expect(portMap[1].Connected).To(BeTrue())
-				Expect(portMap[1].DeviceID).To(Equal(uint(1028)))
-				Expect(portMap[1].VendorID).To(Equal(uint(310)))
-				Expect(portMap[1].ProductName).To(Equal("VVB001"))
-				Expect(portMap[1].Serial).To(Equal("000008512993"))
+				// Debugging: print out the port map including mode, connected, deviceID, vendorID, productName, serial
+				for i, port := range portMap {
+					debugInfo := fmt.Sprintf(
+						"Port %d:\n  Mode: %d\n  Connected: %t\n  DeviceID: %d\n  VendorID: %d\n  ProductName: %s\n  Serial: %s\n",
+						i, port.Mode, port.Connected, port.DeviceID, port.VendorID, port.ProductName, port.Serial,
+					)
+					GinkgoWriter.Write([]byte(debugInfo))
```

golangci-lint (1.61.0)

Error return value of `GinkgoWriter.Write` is not checked

(errcheck)